cart/cart.js

```diff
@@ -1,40 +1,53 @@
-//import { items, cart } from '../data.js';
-import { cart } from '../data.js';
+
+import { items } from '../data.js';
 import { renderTableRow } from './cart-utils.js';
-//import { findById } from '../utils.js';
-//import { calculateTotal } from '../data:cart.js';
+import { findById, getFromLocalStorage, CART } from '../utils.js';
+
+
 const table = document.querySelector('tbody');
+const orderButton = document.querySelector('button');
 
+const cart = getFromLocalStorage(CART) || [];  
 for (let i = 0; i < cart.length; i++) {
-    const items = cart[i];
+    const item = cart[i];
```

Comment:
great work implementing this

Reply:
thank you did you see my notes why is my cart page not linked on git pages.. And the third test I am not sure how to test the function..

Reply:
Hey Sarah, I think Dani is going to go over tomorrow how to properly link and route pages, since it seems to be a common issue with everyone on how to properly deploy.  Also, which test are you referring to? If you want to stop into the TA room, I'm sure we'd be happy to walk you through troubleshooting it. :)